CriticalMaps.xcodeproj/project.pbxproj

```diff
@@ -1273,7 +1273,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CURRENT_PROJECT_VERSION = 19;
-				DEVELOPMENT_TEAM = 5YLLXUZBZ2;
+				DEVELOPMENT_TEAM = P33TK2RVAJ;
```

Comment:
Can you reset project.pbxproj? The changes doesn't seem to be related to the bug fix